1-development/list_posts_by_rating/README.md

```diff
@@ -0,0 +1 @@
+# list_posts_by_rating
```

Comment:
Hi! I will be asking some questions in this PR.

Can you add some info on what I have to setup so I can run this on my machine?
And what database have you used?

Reply:
Hi,

I updated the README.md file with installation steps.
I'm using the MySQL database.

Reply:
Also, can you specify the stack you've used to solve this challenge?

Reply:
I updated the README.md file with the technologies used